pinot-minion/src/main/java/org/apache/pinot/minion/executor/PinotTaskExecutorFactory.java

```diff
@@ -26,7 +26,7 @@
   /**
    * Initializes the task executor factory.
    */
-  void init(MinionTaskZkMetadataManager zkMetadataManager);
+  void init(MinionTaskZkMetadataManager zkMetadataManager, int endReplaceSegmentsSocketTimeoutMs);
```

Comment:
not a big fan of changing the interface here as this interface is not just used by merge and roll up task. Is it possible to put it inside the zkMetadataManager?

Reply:
Updated to store the value in `zkMetadataManager`